src/ProjectTemplates/BlazorTemplates.Tests/playwrightSettings.json

```diff
@@ -28,7 +28,7 @@
     },
     "Firefox": {
       "BrowserKind": "Firefox",
-      "IsEnabled": true
+      "IsEnabled": false
```

Comment:
Can you turn this off on `playwrightSettings.ci.json`

Reply:
Sure if this works I'll switch it to that file, just trying to make things not blow up on linux

Reply:
This was already set in ci.json, so I'll just try removing it and seeing if it blows up again

Reply:
Doesn't look like this works (at least not on helix right now), so I'm just going to turn it off globally